src/xpcc/architecture/platform/devices/stm32/stm32f072-c_r_v-8_b.xml

```diff
@@ -51,7 +51,7 @@
       <vector position="30" name="CEC_CAN"/>
       <vector position="31" name="USB"/>
     </driver>
-    <driver type="adc" name="stm32f0"/>
+<!--     <driver type="adc" name="stm32f0"/> -->
```

Comment:
So in that world, the `name` would be redundant, and all drivers inside the `adc` folder would be checked for device support and any would then be built.
In that world, the `generic` driver folder is just another driver, and not a magic folder hardcoded in the build system ;-P